native/src/components/__tests__/DashboardNavigationTiles.spec.tsx

```diff
@@ -1,15 +1,21 @@
 import { mocked } from 'jest-mock'
 import React from 'react'
-import TestRenderer from 'react-test-renderer'
 
-import { TileModel } from 'shared'
 import { CityModel, LanguageModelBuilder } from 'shared/api'
 
 import buildConfig from '../../constants/buildConfig'
+import render from '../../testing/render'
 import DashboardNavigationTiles from '../DashboardNavigationTiles'
-import NavigationTiles from '../NavigationTiles'
 
 jest.mock('react-i18next')
+jest.mock('react-native-safe-area-context', () => {
+  const actual = jest.requireActual('react-native-safe-area-context')
+  return {
+    ...actual,
+    useSafeAreaInsets: () => ({ top: 0, bottom: 0, left: 0, right: 0 }),
+    SafeAreaProvider: ({ children }: { children: React.ReactNode }) => children,
+  }
+})
```

Comment:
I needed to add it because of the error `Error: No safe area value available. Make sure you are rendering <SafeAreaProvider> at the top of your app.`

Reply:
Mocking `useSafeAreaInsets` should be enough as that is the only thing used by `NavigationTiles`, `SafeAreaProvider` is only used in `App.tsx`. Alternatively, you can also directly use the mock provided by `react-native-safe-area-context`, see e.g. `MalteHelpFormOffer.spec.tsx`.

Reply:
@steffenkleinle thanks, good to know :)